personio-personnel-data-api-oa3.yaml

```diff
@@ -3983,3 +4105,700 @@ components:
+          items:
+            $ref: "#/components/schemas/PublicReportData"
+
+    PublicReportResponse:
+      type: object
+      properties:
+        success:
+          type: boolean
+        metadata:
+          $ref: "#/components/schemas/PublicReportMetaData"
+        offset:
+          type: integer
+        limit:
+          type: integer
+        data:
+          type: array
+          items:
+            $ref: "#/components/schemas/PublicReportItemsData"
+
+    PublicListColumnsResponse:
+      type: object
+      properties:
+        success:
+          type: boolean
+        metadata:
+          $ref: "#/components/schemas/PublicReportMetaData"
+        offset:
+          type: integer
+        limit:
+          type: integer
+        data:
+          $ref: "#/components/schemas/PublicColumnData"
+
+    PublicReportMetaData:
+      type: object
+      properties:
+        total_elements:
+          type: integer
+        current_page:
+          type: integer
+        total_pages:
+          type: integer
+
+    PublicReportData:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "Report"
+        attributes:
+          $ref: "#/components/schemas/PublicReportAttributes"
+
+    PublicReportItemsData:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "Report"
+        attributes:
+          $ref: "#/components/schemas/PublicReportItemsAttributes"
+
+    PublicColumnData:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "Column"
+        attributes:
+          type: array
+          items:
+            $ref: "#/components/schemas/PublicColumnAttributes"
+
+    PublicColumnAttributes:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: first_name
+        human_readable:
+          type: string
+          example: First Name
+        data_type:
+          type: string
+          example: "TEXT"
+
+    PublicReportAttributes:
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/UniqueIdentifier"
+        name:
+          type: string
+          example: Employee vacations past month
+        description:
+          type: string
+          example: Shows vacations of all employees that were taken past month
+        author_first_name:
+          type: string
+          description: Report author
+          example: Robert
+        author_last_name:
+          type: string
+          description: Report author
+          example: Sirano
+        type:
+          type: string
+          example: point_in_time
+          enum:
+            - point_in_time
+            - historical_data
+            - timeframe
+        employee_status:
+          type: string
+          example: currently_active
+          enum:
+            - currently_active
+            - active_at_given_date
+            - all
```

Comment:
What does all mean in this context?

Reply:
@raul-mdelfin, these are the returned options values for this field.